nullaway/src/main/java/com/uber/nullaway/generics/TypeSubstitutionUtils.java

```diff
@@ -1,9 +1,18 @@
 package com.uber.nullaway.generics;
```

Comment:
What kind of substitution are we referring to here exactly? And why are nullability annotations from type variables lost after performing a substitution - Could you clarify more in the diff description?

Reply:
PR description is updated